src/calendar-list/index.js

```diff
@@ -200,13 +201,18 @@ class CalendarList extends Component {
         initialScrollIndex={this.state.openDate ? this.getMonthIndex(this.state.openDate) : false}
         getItemLayout={this.getItemLayout}
         scrollsToTop={this.props.scrollsToTop !== undefined ? this.props.scrollsToTop : false}
+        viewabilityConfig={this.viewabilityConfig}
       />
     );
   }
 }
 
 CalendarList.defaultProps = {
-  calendarHeight: 360
+  calendarHeight: 360,
+  calendarWidth: width,
+  viewabilityConfig: {
+    viewAreaCoveragePercentThreshold: 100,
```

Comment:
So this would mean the list item would have to take up 100% of the viewport - Wouldnt we want something more along the lines of

```
itemVisiblePercentThreshold: 25
```
Which would mean the FlatList item would determine it is in view after 25% of it is visible?

I may be misinterpreting viewabilityConfig, but wanted to ensure the right default prop here

Reply:
I left the default prop to 100, since up until now it was taking the full viewport, and left the opportunity for developers to decide whether they want to change that. Furthermore, I didn't spot that issue with CalendarList's without the pagination enabled. (`pagingEnabled` is valid prop only for horizontal lists), therefore I decided to leave it as it was with the opportunity to change on the fly. Also the behaviour of firing up the event is different with the threshold, in my case it was not fitting having 25, therefore the suggestion for 50 in my comment. But as I said, its up to the developer to decide if/how to use it. Open for changes if requested tho. )

Reply:
Sounds good!

Reply:
I think it's best to remove this default so it will be passed only if the user pass it, otherwise, we leave the RN defaults.